recipe/meta.yaml

```diff
@@ -90,7 +90,7 @@ outputs:
 
     requirements:
       run:
-        - python {{ python_min }}
+        - python >={{ python_min }}
```

Comment:
In favor of also specifying an upper bound here, currently this communicates that the examples work with 3.13 (which we have probably not tested), 3.14 (moreso), etc.

Reply:
Good call, thanks!